teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>How to get <s>intersting</s> <strong>interesting</strong> properties <s>from</s> based on aPubChem CID </blockquote><blockquote>We can get <s>intersting</s> <strong>interesting</strong> properties for <strong>a </strong>compound through <s>it's</s> <strong>its</strong> PubChem CID, such as molecular weight, pKd, logP, etc. Here, we will search for the molecular weight for Aspirin.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done</p>